### DISCUSSION TITLE
Picker dialog not keep opening when selected or click on it

### DISCUSSION THREAD

**Discussion by phucsang0spt** [gpbl/react-day-picker#1098] [Support]
I use DayPickerInput to trigger picker dialog to selected date but When I click anywhere on the dialog it will auto close dialog and onDayChange not fire even I selected a date
 
```
<DayPickerInput
            placeholder="YYYY/MM/DD"
            format="YYYY/MM/DD"
            onDayChange={date => {
              console.log(date);
            }}
</DayPickerInput>
```

**Comment by jeslage** [comment]
@phucsang0spt have you found a solution for this issue? I am currently having the same problem with the dialog.
